server/chat-plugins/youtube.ts

```diff
@@ -24,10 +24,12 @@ export class YoutubeInterface {
 	interval: NodeJS.Timer | null;
 	intervalTime: number;
 	data: AnyObject;
```

Comment:
I'd probably want this to be strongly typed if you're going to do something like this.